### DISCUSSION TITLE
XUbuntu 20.04 mouse pointer is not moving in the clients

### DISCUSSION THREAD

**Discussion by ale3andro** [epoptes/epoptes#126] [Q&A]
Since I have upgraded my lab Computers to XUbuntu 20.04, the mouse pointer is stuck at the center of the clients screens (when I broadcast my screen) but it does not move. The students can watch my screen but they can't see where is my mouse and where I click.
If there an option to change in order to make it work as it did in previous versions?

**Comment by alkisg** [comment]
Which vnc viewer are you using? E.g. `ps faux | grep vnc` on the client, while broadcast is running, will tell you.

In the [Greek Schools PPA](https://launchpad.net/~ts.sch.gr/+archive/ubuntu/ppa) I've uploaded an older version of xvnc4viewer, which is still the best VNC viewer even though it was removed from Debian.
And another option is to download the proprietary .deb from there: https://www.realvnc.com/en/connect/download/viewer/. That one even supports scaling.

After installing the new VNC viewer, remove the old one, e.g. tiger or tight VNC.

**Reply by ale3andro** [reply]
ps faux | grep vnc gives the following
ale3and+   16714 16.0  0.4 260868 19456 ?        Sl   10:56   0:00          |   |   |   \_ x11vnc -noshm -nopw -quiet -viewonly -shared -forever -nolookup -24to32 -threads -rfbport 48763 -rfbauth /home/ale3andro/.config/epoptes/vncpasswd

I will try to install the proprietary deb. I have noticed that while in broadcast the vnc viewer in the clients randomly crashes and the broadcast stops. In the older version of Xubuntu LTS, I have never experienced this issue. Maybe this package will fix this issue too.
Thank you very much

**Comment by ale3andro** [comment]
The proprietary deb VNC Viewer does not solve the issue. The mouse pointer is still not shown.
The older xvnc4viewer from the ts.sch.gr repo does work and solves this issue.
Thank's again.

**Comment by alkisg** [comment]
> ps faux | grep vnc gives the following

I guess that's on the server, while we wanted the output from the clients.

OK great, problem solved; for the best future results, someone should file a bug against tiger/tightvnc.

**Reply by ale3andro** [reply]
Yes it was.
The client used tiger vnc viewer

**Comment by alkisg** [comment]
I reported an issue against tigervnc back in 2017, it hasn't been solved yet:
https://github.com/TigerVNC/tigervnc/issues/400

People that want to help could contribute directly there. Until it's fixed, I recommend xvnc4viewer and RealVNC.
